Resources/public/css/views/dashboard/dashboard.css

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview * {
+    box-sizing: border-box;
+}
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview > .pure-u {
```

Comment:
This is required to remove ugly margin from the top of the view container. When we remove PureCSS then we'll remove this line.